content/repositories/creating-and-managing-repositories/creating-a-template-repository.md

```diff
@@ -25,7 +25,14 @@ To create a template repository, you must create a repository, then make the rep
 
 After you make your repository a template, anyone with access to the repository can generate a new repository with the same directory structure and files as your default branch. They can also choose to include all the other branches in your repository. Branches created from a template have unrelated histories, so you cannot create pull requests or merge between the branches. For more information, see "[AUTOTITLE](/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template)."
 
+{% note %}
+
+**Note:** You can use a template repository as starter code for an assignment on {% data variables.product.prodname_classroom %}. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-an-assignment-from-a-template-repository)."
+
+{% endnote %}
```

Comment:
```suggestion
{% ifversion fpt %}
{% note %}

**Note:** You can use a template repository as starter code for an assignment on {% data variables.product.prodname_classroom %}. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-an-assignment-from-a-template-repository)."

{% endnote %}
{% endif %}
```

Adding this versioning in should fix that broken link! Basically, the article you're adding this note to, "Creating a template repository", is versioned for Free, Pro, & Teams (FPT) and GitHub Enterprise Cloud (GHEC), and the article you're linking to in the note is versioned for only FPT, so when someone is reading the GHEC version of "Creating a template repository", then clicks the link in the note, the link points to a non-existent version of "Create an assignment from a template repository."

Reply:
Oh I was wondering why the link was breaking, thank you!